mxcube3/ui/components/SampleView/DrawGridPlugin.js

```diff
@@ -208,10 +214,9 @@ export default class DrawGridPlugin {
     canvas.renderAll();
   }
 
-  heatMapColorforValue(gd, value) {
-    const h = (1.0 - value) * 240;
-    const dataFill = `hsla(${h}, 100%, 50%, ${this.overlayLevel})`;
-
+  heatMapColorForValue(gd, value) {
+    let dataFill = `rgba(${parseInt(value[0], 10)}, ${parseInt(value[1], 10)},`;
+    dataFill += `${parseInt(value[2], 10)}, ${this.overlayLevel})`;
```

Comment:
but then the color coding comes from the data sent by the server, you do not expect a single score value

Reply:
I think we already discussed this, and we had some sort of agreement. So the grid results have always been given as R,G,B values and thats what we need to adapt to today. We have added a fourth field in the results so that its now R,G,B,score to fulfill your needs. Where score is given by the analysis software, handling of the score value needs to to be added if needed. We can have a discussion on migrating to only basing the the results on sore for version 3.1.